sarama/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/SkyAPM/go2sky-plugins/sarama
```

Comment:
Recommended that `sarama` be changed to `kafka-reporter`, what do you think? @matianjun1